paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -1516,6 +1536,61 @@ class PaymentSheet internal constructor(
         }
     }
 
+    /**
+     * Options to block certain card brands on the client
+     */
+    @ExperimentalCardBrandFilteringApi
+    sealed class CardBrandAcceptance : Parcelable {
+
+        /**
+         * Card brand categories that can be allowed or disallowed
+         */
+        @Parcelize
+        enum class BrandCategory : Parcelable {
```

Comment:
You'll need to add `@ExperimentalCardBrandFilteringApi` to all of these nested classes as well.

Reply:
Thanks, updated the API review doc with those annotations, lmk if looks good and will add in the PR if so.